.eslintrc.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
     'flowtype/define-flow-type': ERROR,
     'flowtype/no-mutable-array': ERROR,
     'flowtype/no-types-missing-file-annotation': ERROR,
-    'flowtype/require-exact-type': [ERROR, ALWAYS],
+    'flowtype/require-exact-type': OFF,
```

Comment:
Flow is already set to `exact_by_default` to `true` in `.flowconfig`, so this was redundant syntax {| |}